common/recommender/Baseline.js

```diff
@@ -35,11 +35,13 @@ class Baseline {
   extractFeatures(entry) {
     const urlObj = URL(entry.url);
     const host = urlObj.host;
+    // For empty profiles.
     const occurrences = this.domainCounts.get(host) || 1;
+    const domainCountsSize = this.domainCounts.size || 1;
+    const tf = entry.visitCount || 1;
+    const idf = Math.log(domainCountsSize / occurrences) || 1;
```

Comment:
@ncloudioj the value `1` can only occur if you only visit a single domain which seems difficult to achieve but smoothing makes sense.